vignettes/GLM_intro_course.Rmd

```diff
@@ -2167,7 +2231,7 @@ testDispersion(r2, plot = FALSE)  ## you can change options to test for underdis
 * there are specific solutions if the origin is zero-augmentation
 
 
-## Modeling simply overdispersion
+## Modeling simple overdispersion
```

Comment:
I meant simply as in a simple way of modelling overdispersion.

Reply:
Ahh yea I see. 'Modeling simply overdispersion' is not grammatically correct, but I think 'Simple modeling of overdispersion' (which is actually used in the next slide!) would be good.

Reply:
do that